anta/cli/nrfu/utils.py

```diff
@@ -122,6 +124,21 @@ def print_jinja(results: ResultManager, template: pathlib.Path, output: pathlib.
             file.write(report)
 
 
+def save_markdown_report(ctx: click.Context, md_output: pathlib.Path, *, only_failed_tests: bool = False) -> None:
+    """Save the markdown report to a file.
+
+    Parameters
+    ----------
+        ctx: Click context containing the result manager.
+        md_output: Path to save the markdown report.
+        only_failed_tests: If True, only failed tests will be included in the report. Default is False.
+    """
+    console.print()
+    MDReportGenerator.generate(results=_get_result_manager(ctx), md_filename=md_output, only_failed_tests=only_failed_tests)
```

Comment:
This could fail and raise an OSError, in that case we would need to catch it cleanly and not print "Markdown report saved"

Reply:
Fixed in 5c449949773cb86a3c25aa336bd3d83f5150773c